src/api/cpp/cvc5_kind.h

```diff
@@ -289,46 +322,56 @@ enum Kind : int32_t
   /**
    * Application of an uninterpreted function.
    *
-   * Parameters: n > 1
-   *   - 1: Function Term
-   *   - 2..n: Function argument instantiation Terms
+   * - Arity: `n > 1`
+   *   - `1:` Function Term
+   *   - `2..n:` Function argument instantiation Terms of any first-class Sort
    *
-   * Create with:
-   *   - `Solver::mkTerm(Kind kind, const Term& child1, const Term& child2) const`
-   *   - `Solver::mkTerm(Kind kind, const Term& child1, const Term& child2, const Term& child3) const`
-   *   - `Solver::mkTerm(Kind kind, const std::vector<Term>& children) const`
+   * - Create Term of this Kind with:
+   *   - Solver::mkTerm(Kind, const std::vector<Term>&) const
+   *   - Solver::mkTerm(const Op&, const std::vector<Term>&) const
+   *
+   * - Create Op of this kind with:
+   *   - Solver::mkOp(Kind) const
    */
   APPLY_UF,
 #if 0
   /* Boolean term variable */
   BOOLEAN_TERM_VARIABLE,
 #endif
   /**
-   * Cardinality constraint on uninterpreted sort S.
-   * Interpreted as a predicate that is true when the cardinality of S
-   * is less than or equal to the value of the second argument.
+   * Cardinality constraint on uninterpreted sort.
    *
-   * Parameters:
-   *   - 1: Term of sort S
-   *   - 2: Positive integer constant that bounds the cardinality of sort S
+   * Interpreted as a predicate that is true when the cardinality of
```

Comment:
This is no longer accurate, this can only be created via `mkCardinalityConstraint`.

I can fix this in a separate PR.

Reply:
Does this only concern the documentation? Then I can fix it here.

Reply:
There are quite a few things to fix as a result, I can just do this separately.

Reply:
(Although I would keep the changes you have for this Kind, as they are an improvement).